election-ui-javascript/src/dataFetcher/index.test.js

```diff
@@ -10,12 +10,20 @@ const mockFakeApi = () => {
       results: [
         {
           'party': 'Independent',
-          'candidate': 'Lord Buckethead',
-          'votes': '9900'
+          'votes': '9900',
+          'candidateId': 1
```

Comment:
this doesn't match what comes from the fakeAPI, not sure if it is intentional or not.

Reply:
I think it may be intentional, if so we need all assessors to be aware of it, if not intentional thanks for the fix.

Reply:
I think this test data is meant to match that from the API, so I would consider this a _mistake_ and suggest that we merge this change.